res/database/Default/user/ronny.xml

```diff
@@ -695,7 +695,7 @@ Illustrious guests and music by [3|Nick].</en>
 				- Interesse an Liebesfilmen steigern
 			-->
 			<effects>
-				<effect trigger="happen" type="modifyPersonPopularity" referenceGUID="21e88c65-0cc6-4bd1-bd14-b48b6ce25402" valueMin="0.1" valueMax="0.2" />
+				<effect trigger="happen" type="modifyPersonPopularity" referenceGUID="${actorID}" valueMin="0.1" valueMax="0.2" />
```

Comment:
Ist zwar an sich nicht wirklich wichtig, aber evtl statt "actorID" die Variable "actorGUID" nennen, einfach ... der Korrektheit wegen ;-)

Reply:
Die Änderungen in der Datenbank werden sowieso vor dem Merge zurückgebaut. Sind nur für's lokale Testen der neuen Funktionen.